pom.xml

```diff
@@ -229,7 +229,7 @@
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
             <version>5.1.46</version>
-            <scope>test</scope>
+            <!--<scope>test</scope>-->
```

Comment:
Reset this to test scope again. 

In our python module, add the logic that we can specify the extra classpath. Let the test script use that logic with the pre-downloaded jars.

Reply:
I'm not sure how to do this. How could I specify a specific JDBC driver when starting the JVM?

Reply:
Use the classpath option here: https://www.py4j.org/py4j_java_gateway.html#py4j.java_gateway.JavaGateway.launch_gateway